doc/sphinx/conf.py

```diff
@@ -147,7 +147,7 @@
 #html_theme_options = {'collapsiblesidebar': True}
 
 # Activate the theme.
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+# html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
Deprecated since v0.2.5 and we're on v1.3 (sphinx) and v3.0 (next-sphinx):

https://github.com/Pennsieve/sphinx_rtd_theme/blob/c9145b48657d88f4fc19dd23e57284fc186c0a76/README.rst#L51

Started raising an error in v8 of sphinx so removing / commenting out